app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -11,6 +11,8 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       default_hostname: '',
       amqp_hostname: '',
       hawkular_hostname: '',
+      provider_options_values: {},
```

Comment:
Not quite sure what these are, but looks like they are generic options, not related to the specific instance you're editing, is that so?

If so, you probably want `$scope.somethingElse` :) - the idea is that the `model` object should be just the data of that particular instance (not necessarily quite 1:1 with the database, but as close as possible, so that you can just post the whole thing to the server, and not have to delete fields when saving.).